src/sentry/relay/config/__init__.py

```diff
@@ -410,7 +410,7 @@ def _filter_option_to_config_setting(flt, setting):
         "d:transactions/measurements.lcp@millisecond",
         "d:transactions/measurements.app_start_cold@millisecond",
         "d:transactions/measurements.app_start_warm@millisecond",
-        "d:transactions/measurements.cls@millisecond",
+        "d:transactions/measurements.cls@none",
```

Comment:
CLS is unitless, see https://web.dev/cls/.

Reply:
https://github.com/getsentry/sentry/blob/be6c69a31f2343a870b5e58a5c0cf253f2bb52f4/src/sentry/snuba/metrics/naming_layer/mri.py#L34-L34 we have TODO here for better check `unit`.